python/paddle/incubate/autograd/primrules.py

```diff
@@ -132,69 +132,21 @@ def linear_jvp(op, *args, **kwargs):
 def elementwise_add_orig2prim(op, x, y):
     if x.shape != y.shape:
         y = broadcast(y, shape=x.shape)
```

Comment:
这里为什么删除这段逻辑呢？

Reply:
This is basic elementwise test, which doesn't have scale arguments anymore, so that if will never happen